extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/mongodb/runtime/LiquibaseMongodbRecorder.java

```diff
@@ -18,11 +20,11 @@ public class LiquibaseMongodbRecorder {
 
     public Supplier<LiquibaseMongodbFactory> liquibaseSupplier(LiquibaseMongodbConfig config,
             LiquibaseMongodbBuildTimeConfig buildTimeConfig, MongodbConfig mongodbConfig) {
-        return new Supplier<LiquibaseMongodbFactory>() {
-            @Override
-            public LiquibaseMongodbFactory get() {
-                return new LiquibaseMongodbFactory(config, buildTimeConfig, mongodbConfig.defaultMongoClientConfig);
-            }
+        return () -> {
```

Comment:
There's a good reason we don't use lambdas in runtime code as they require more memory. Please stick to the preexisting pattern.